sources/models/src/lib.rs

```diff
@@ -127,6 +128,10 @@ struct KubernetesSettings {
     authentication_mode: KubernetesAuthenticationMode,
     bootstrap_token: KubernetesBootstrapToken,
     standalone_mode: bool,
+    eviction_hard: HashMap<EvictionHardKey, EvictionHardValue>,
+    kube_reserved: HashMap<ReservedResourcesKey, ReservedResourcesValue>,
+    cpu_manager_policy: CpuManagerPolicy,
+    allowed_unsafe_sysctls: Vec<SingleLineString>,
```

Comment:
By reading  `allowed_unsafe_sysctls` as SingleLineString we are allowing customer to configure any `sysctl` parameters they would like. There can be lots of `sysctl` parameters and I guess we would not like to support all of them. I think it was discussed that we would like to start by asking customer what `sysctl` parameter they would like to configure and evaluate the consequence of supporting those parameters.

Reply:
I discussed with Erikson before, and we thought we shouldn’t dictate this setting. Everyone might have different policies towards sysctl values. so right now we just make sure this variable is configurable. I'm going to discuss with more people to see If I am ok here.    Thanks

Reply:
We're already allowing users to set sysctl values via corndog. I don't think we should gatekeep users from deciding what sysctls to enable for their k8s pods.

Reply:
That being said, I do think it's worth calling out that unsafe sysctls are use-at-your-own-risk in the docs. The [k8s documentation](https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#setting-sysctls-for-a-pod) has a warning for setting unsafe sysctls.